config/environments/development.rb

```diff
@@ -30,6 +30,8 @@
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
+  # Sometimes we want to test amazon
+  # config.active_storage.service = :amazon
```

Comment:
Rather than including this this way, can you just expand the L31 note to say "Change to `:amazon` to test S3 storage" or something?